services/blockchain-service/blockchain-service-base.js

```diff
@@ -41,14 +41,19 @@ class BlockchainServiceBase {
       ...args
     ).encodeABI();
 
+    let gasPrice = blockchain.name === "otp"
+        ? await this.web3.eth.getGasPrice()
+        : this.web3.utils.toWei("100", "Gwei");
+
+    if(blockchain.name==="otp") {
```

Comment:
there is no need to have two checks for blockchain name. We can have gas price calculation and multiplication in a single if clause

Reply:
good point